.prettierrc.cjs

```diff
@@ -2,4 +2,12 @@
 
 module.exports = {
   singleQuote: true,
+  overrides: [
+    {
+      files: ['*.js', '*.ts', '*.cjs', '.mjs', '.cts', '.mts', '.cts'],
```

Comment:
`.cts` is in here twice, and it seems some entries are missing the `*` character?

Reply:
Ope